iceoryx_examples/automotive_soa/include/minimal_skeleton.hpp

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2022 by Apex.AI Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef IOX_EXAMPLES_AUTOMOTIVE_SOA_MINIMAL_SKELETON_HPP
+#define IOX_EXAMPLES_AUTOMOTIVE_SOA_MINIMAL_SKELETON_HPP
+
+#include "topic.hpp"
+
+#include "owl/kom/event_publisher.hpp"
+#include "owl/kom/field_publisher.hpp"
+#include "owl/kom/method_server.hpp"
+#include "owl/types.hpp"
+
+class MinimalSkeleton
+{
+  public:
+    static constexpr char m_serviceIdentifier[] = "MinimalSkeleton";
```

Comment:
See other comments, make it a global and move it to `owl/globals.hpp`

Reply:
See my answer in the [previous comment](https://github.com/eclipse-iceoryx/iceoryx/pull/1355#discussion_r886603361).